robotswerve.py

```diff
@@ -121,8 +128,12 @@ def robotPeriodic(self):
 
         self.intake_command_scheduler.run()
 
-        #self.vision.getCamEstimate()
-        #self.vision.showTargetData()
+        if self.vision is not None:
+            try:
+                self.vision.getCamEstimates(specificTagId=lambda: self.alignmentTagId)
+                self.vision.showTargetData()
+            except Exception:
+                print("Unable to retrieve vision pose estimates")
```

Comment:
This exception should be gated. Else it may spam and cause buffer print overruns.

Reply:
Also it is good to do Exception as e
then log some data from e with the error. Else we won't know why it failed.